src/ui/atoms/inputBase/InputBase.tsx

```diff
@@ -0,0 +1,61 @@
+// This rule is deactivated because we can not add readonly on the entire ChangeEvent
+/* eslint-disable @typescript-eslint/prefer-readonly-parameter-types */
+import React from 'react'
+import type { RefObject } from 'react'
+import classNames from 'classnames'
+import './inputBase.scss'
+
+export type InputBaseProps = Readonly<{
+  /**
+   * Placeholder text.
+   */
+  readonly placeholder?: string
+  /**
+   * Defines the callback called when the input value changes.
+   */
+  readonly onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void
+  /**
+   * The value of the `input` element, required for a controlled component.
+   */
+  readonly value?: string | string[]
```

Comment:
I don't see the point of having this kind of type. Can you explain?

Reply:
@ccamel we need a simple `string` for a select with single selection, but also the possibility to have `string[] `for the multi-selection.

Reply:
So sorry to bother you with this, but I still don't understand what do you mean by single/multi selection for an [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) html element. Could you elaborate or refer me to a document that explains this point? Thx ☺️

Reply:
@lolottetheclash ok after exchanging together I see the point. I'm not excited about it, but hey, it's okay for me. 😄